{
  "type": "class",
  "name": "TestRuntimeException",
  "qualified_name": "net.minecraft.test.TestRuntimeException",
  "package": "net.minecraft.test",
  "modifiers": [],
  "annotations": [],
  "description": "",
  "inheritance": {
    "extends": null,
    "implements": [],
    "inheritance_tree": []
  },
  "nested_classes": [],
  "fields": [],
  "constructors": [
    {
      "name": "TestRuntimeException",
      "modifiers": [],
      "parameters": [],
      "description": "",
      "deprecated": false
    }
  ],
  "methods": [],
  "enum_constants": [],
  "since": null,
  "deprecated": false,
  "source_file": "Frameworks/1.15.2/net/minecraft/test/TestRuntimeException.html"
}